src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -233,18 +225,15 @@ const SearchFieldList = ({ onSearch }: Props) => {
       setCountries(countries.map((c) => c.Name));
       setDepartments(departments);
       setBuildings(buildings);
-
       setInvolvements(involvements);
       setLoading(false);
     };
 
     loadPage();
-
   }, []);
   useEffect(() => {
-    // Read search params from URL on navigate (including first load)
-    if (shouldReadSearchParamsFromURL.current) {
-      const newSearchParams = deserializeSearchParams(new URLSearchParams(location.search));
+    const readSearchParamsFromURL = () => {
```

Comment:
Why is this code changing in this PR?

Reply:
After a git pull I kept getting errors in this file, so I used the file from develop then made my necessary header changes to it